doc/debian/jitsi-meet/jitsi-meet.example

```diff
@@ -39,6 +39,15 @@ server {
     index index.html index.htm;
     error_page 404 /static/404.html;
 
+    gzip on;
+    gzip_types text/plain text/css application/javascript;
```

Comment:
Could you add application/json for the translation files?

Reply:
I've added that, should be in the pull request now.  Thanks for the suggestion, I didn't think about the translation files!